www/README.md

```diff
@@ -66,65 +48,53 @@ in [Markdown](https://en.wikipedia.org/wiki/Markdown).
 .
 ├── Makefile    # contains helpers to quickly start up the development environment
 ├── README.md
-├── chef-sh     # hidden directory that contains the pre-compiled source for chef.sh
-├── chef-www    # chef/chef-www submodule to pull in the chef-sh theme for local development
-├── site        # the hugo site directory used for local development
+├── www        # the hugo site directory used for local development
 ```
 
 ### Local Content
 ```
 .
 ├── site
 │   ├── content
-│   │   ├── docs
-│   │   │   ├── chef-workstation    # where to keep markdown file documentation
+│   │   ├── workstation             # where to keep markdown file documentation
 │   ├── data
 │   │   ├── chef-workstation        # where to keep structured data files used for data templates
 │   ├── layouts
 |   │   ├── shortcodes
-|   │   │   ├── cw-*                # how to name your workstation-specific shortcodes
+|   │   │   ├── workstation         # how to name your workstation-specific shortcodes
 |   ├── static
 |   |   ├── images
 |   |   |   ├── chef-workstation    # where to keep any images you need to reference in your documentation
+|   |   ├── css
 ```
 
 ### What is happening behind the scenes
 
-Everytime a PR is merged into `chef/chef-workstation`, the contents of the `chef-workstation` directories from the `site` directory are synced to `chef/chef-www` and the `chef-www-acceptance.cd.chef.co` acceptance environment is automatically updated. This is handled by the Expeditor subscriptions in the `chef/chef-www` GitHub repository.
+TODO
```

Comment:
@IanMadd I'll need some help with the TODOs here - I'm not sure of the process of how our docs get deployed